docs/rules/prefer-export-from.md

```diff
@@ -0,0 +1,61 @@
+# Prefer `export…from` when re-exporting
+
+When re-exporting from module, it's unnecessary to import and then export, it can be done in one `export…from` declaration.
```

Comment:
```suggestion
When re-exporting from a module, it's unnecessary to import and then export. It can be done in a single `export…from` declaration.
```